test/e2e/tests/backend_tls_settings.go

```diff
@@ -171,7 +171,7 @@ func confirmEchoBackendRes(httpRes *http.ExpectedResponse, expectedResBody *Resp
 			InsecureSkipVerify: true, //nolint:gosec
 		},
 	}
-	req := http.MakeRequest(t, httpRes, gwAddr, "HTTPS", "https")
+	req := http.MakeRequest(t, httpRes, gwAddr, "HTTP", "http")
```

Comment:
Client TLS is not required for the Backend TLS setting test. Removing it to make the test simpler and easier to understand.